include/Game/generalEnemyMgr.h

```diff
@@ -72,7 +72,7 @@ struct GeneralEnemyMgr : public GenericObjectMgr, public CNode {
 	void addEnemyNum(int, u8, Game::GenObjectEnemy*);
 	void getEnemyNum(int, bool);
 	void useHeap();
-	void getEnemyMgr(int);
+	EnemyMgrBase* getEnemyMgr(int);
```

Comment:
For any functions you realise the purpose of, add the variable name to the declaration, this'll ease up the workload later on when we do the documentation phase, as things will already be documented.

In terms of unknowns, leave them empty, but if we know the use of them, put the appropriate variable name

Reply:
noted for future, I agree it's a good way to minimise documentation load later.